### PR TITLE
add_slices

### PR DESCRIPTION
**Pull request by Bodasyuk**
dev
## JIRA

* [Main JIRA ticket](https://jira.softserve.academy/secure/RapidBoard.jspa?rapidView=id)

## Code reviewers

- [ ] @github_username

### Second Level Review

- [ ] @github_username

## Summary of issue

ToDo

## Summary of change

ToDo

## Testing approach

ToDo

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

**Review by petromonka — APPROVED** [review]

File: src/pages/City/City/City.tsx [hunk 1]
```diff
@@ -688,7 +688,7 @@ const City = () => {
             </Title>
             <Row className={members.length >= 4 ? "cityItems1" : "cityItems"} justify="center" gutter={[0, 16]}>
               {members.length !== 0 ? (
-                members.map((member) => (
+                members.slice(0, 9).map((member) => (
```
Comment (VSBalanchuk) [inline src/pages/City/City/City.tsx]: Remove magic number "9"

File: src/pages/City/City/City.tsx [hunk 2]
```diff
@@ -745,7 +745,7 @@ const City = () => {
             </Title>
             <Row className="cityItems" justify="center" gutter={[0, 16]}>
               {admins.length !== 0 ? (
-                admins.map((admin) => (
+                admins.slice(0, 6).map((admin) => (
```
Comment (VSBalanchuk) [inline src/pages/City/City/City.tsx]: Same here

File: src/pages/City/City/City.tsx [hunk 3]
```diff
@@ -808,7 +808,7 @@ const City = () => {
             </Title>
             <Row className="cityItems" justify="center" gutter={[0, 16]}>
               {documents.length !== 0 ? (
-                documents.map((document) => (
+                documents.slice(0, 6).map((document) => (
```
Comment (VSBalanchuk) [inline src/pages/City/City/City.tsx]: And here

**Review by IlliaDR — APPROVED** [review]

File: src/pages/City/City/City.tsx (hunk 3 above)
Comment (VolodymyrKostiv) [inline src/pages/City/City/City.tsx:811]: Why don't you use constant for this?

**Review by VolodymyrKostiv — APPROVED** [review]

**Review by VSBalanchuk — APPROVED** [review]

**Pull request merged**